src/battery-limit.tsx

```
@@ -0,0 +1,49 @@
+import { setBatteryLimit } from "./utils/batt_utils";
+
+interface CommandForm {
+  limit: string;
+}
+
+export default function Command() {
+  const { pop } = useNavigation();
+  
+  async function handleSubmit(values: CommandForm) {
+    try {
+      const limit = parseInt(values.limit, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)